src/server/cdb/data/process.c

```diff
@@ -68,7 +68,7 @@ process_teardown(void)
     }
 
     if (cdb_handle != NULL) {
-        struct CDBHandle *p = cdb_handle;
+        struct cdb_handle *p = cdb_handle;
         cdb_handle = NULL;
         cdb_handle_destroy(p);
     }
```

Comment:
adding these comments here, because github doesn't support inline comments outside of the diff:

line 78: we can just use bstring_init to reset value_buf
like 104: nit: i think you can just say `rsp->vstr = *vstr;`

Reply:
ah, good call!

I wasn't 100% sure about the semantics for doing that assignment on 104 so i just did the safe thing :)